tests/test_image_upload.py

```diff
@@ -74,6 +89,20 @@ def windows_path(path: str):
         yield path
 
 
+# Whenever you modify expected_pass_list, delete files with names from the old list
+# from test_data, set autouse to True and run test_image_upload_file to regenerate files
+@pytest.fixture(autouse=False, scope="module")
```

Comment:
autouse=False is default btw, no need to explicitly specify it.

Reply:
See [below.](#discussion_r1298506966)